bench/analytics/custom_analysis/atlas_analysis.ipynb

```diff
@@ -16,12 +16,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# The test and patch build to use. Change these to produce different plots.\n",
-    "test_name = \"perf-test-medium-multi-lsm\"\n",
+    "test_name = \"perf_test_evict_btree_1\"\n",
```

Comment:
@ajmorton We chatted about renaming the `test_name` offline, and here is the change.